### PR TITLE
boards: arm: nucleo_f030r8: add die-temp0 alias

### PR DESCRIPTION
**Pull request by soburi**
Add die-temp0 alias to die_temp node.
The `die_temp_polling` sample uses this alias.

And prior to that, I added a node for adc1 and enabled the ADC.
Die-temp sensor depends on ADC.

### CODE REVIEW

**Review by gautierg-st — CHANGES_REQUESTED** [review]
Just a minor change, otherwise LGTM

File: samples/sensor/die_temp_polling/boards/nucleo_f030r8.overlay [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2023 TOKITA Hiroshi
+ */
+
+&die_temp {
+	status = "okay";
+};
```
[the remark this answers is not in the document]
Comment (soburi) [reply samples/sensor/die_temp_polling/boards/nucleo_f030r8.overlay]: Thank you for reviewing.

I fixed it.

**Review by gautierg-st — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by FRASTM — COMMENTED** [review]
LGTM
can you please add the log output for the die temperature measurement

**Comment by soburi** [comment]
@FRASTM 

> can you please add the log output for the die temperature measurement

Is my understand that "It should better use LOG_INF rather than printk" is correct?

https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/sensor/die_temp_polling/src/main.c#L37

I want this PR focus on nucleo_f030r8 changes.
I'm thinking of dealing with it in another PR, but is that okay?

**Comment by FRASTM** [comment]
> @FRASTM
> 
> > can you please add the log output for the die temperature measurement
> 
> Is my understand that "It should better use LOG_INF rather than printk" is correct?
> 
> https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/sensor/die_temp_polling/src/main.c#L37
> 
> I want this PR focus on nucleo_f030r8 changes. I'm thinking of dealing with it in another PR, but is that okay?

Sorry, nothing else to change here, my question was about an output log like the following:
```
STM32 Die temperature sensor test                                               
Current temperature: 18.9 �°C 
```
I was wondering if you had run a sample application to demonstrate that the internal temperature sensor
on ADC1 channel 16 is functional with your change.

**Pull request merged**